Translations/pt_BR.po

```diff
@@ -4391,34 +4357,24 @@ msgid "Activate to read"
 msgstr "Ative para ler"
 
 #: Source/items.cpp:1844
-#, fuzzy
-#| msgid "Activate to view"
 msgid "{} to view"
-msgstr "Ative para visualizar"
+msgstr "{} para visualizar"
 
 #: Source/items.cpp:1850
-#, fuzzy
-#| msgid "{:+d} to strength"
 msgid "{} to use"
-msgstr "{:+d} para força"
+msgstr "{} para usar"
 
 #: Source/items.cpp:1853
-#, fuzzy
-#| msgid ""
-#| "Select from spell book, then\n"
-#| "cast spell to read"
 msgid ""
 "Select from spell book,\n"
 "then {} to read"
 msgstr ""
-"Selecionar feitiço atual, então\n"
-"lance o feitiço para ler"
+"Selecionar feitiço atual, \n"
```

Comment:
"Selecionar" is a poot tense choice here and clashes with the second part of the sentence. Also doesn't match the beginning.

Suggestion:
"Selecione do livro de feitiços, \n"